[0m15:19:55.686024 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10704ba10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1070a9d50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1070aa5d0>]}


============================== 15:19:55.688495 | 32e1d105-96c7-41e8-b30c-6d2df1a3422b ==============================
[0m15:19:55.688495 [info ] [MainThread]: Running with dbt=1.7.2
[0m15:19:55.688795 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': '/Users/darshanajit/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m15:20:06.033938 [error] [MainThread]: Encountered an error:

[0m15:20:06.039008 [error] [MainThread]: Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/cli/requires.py", line 90, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/cli/requires.py", line 75, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/cli/main.py", line 515, in init
    results = task.run()
              ^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/task/init.py", line 322, in run
    project_name = self.get_valid_project_name()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/task/init.py", line 276, in get_valid_project_name
    name = click.prompt("Enter a name for your project (letters, digits, underscore)")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m15:20:06.054165 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 10.41125, "process_user_time": 0.865268, "process_kernel_time": 0.143374, "process_mem_max_rss": "109035520", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m15:20:06.055006 [debug] [MainThread]: Command `dbt init` failed at 15:20:06.054802 after 10.41 seconds
[0m15:20:06.055565 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1070a9f90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x102432890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1024327d0>]}
[0m15:20:06.056101 [debug] [MainThread]: Flushing usage events
[0m15:20:12.679335 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10983e310>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109858910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10986e410>]}


============================== 15:20:12.681366 | 5c1bada1-b178-4d34-8fb5-af32fcc5d018 ==============================
[0m15:20:12.681366 [info ] [MainThread]: Running with dbt=1.7.2
[0m15:20:12.681662 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/Users/darshanajit/.dbt', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'invocation_command': 'dbt init', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:20:34.731552 [debug] [MainThread]: Starter project path: /opt/homebrew/lib/python3.11/site-packages/dbt/include/starter_project
[0m15:20:34.743627 [info ] [MainThread]: 
Your new dbt project "airflow_dbt_poc" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m15:20:34.744402 [info ] [MainThread]: Setting up your profile.
[0m15:32:23.399489 [error] [MainThread]: Encountered an error:

[0m15:32:23.422814 [error] [MainThread]: Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/cli/requires.py", line 90, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/cli/requires.py", line 75, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/cli/main.py", line 515, in init
    results = task.run()
              ^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/task/init.py", line 344, in run
    self.setup_profile(profile_name)
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/task/init.py", line 264, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/task/init.py", line 181, in create_profile_from_target
    self.create_profile_from_profile_template(profile_template, profile_name)
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/task/init.py", line 165, in create_profile_from_profile_template
    target = self.generate_target_from_input(prompts, initial_target)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/dbt/task/init.py", line 131, in generate_target_from_input
    target[key] = click.prompt(
                  ^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m15:32:23.429708 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 730.78937, "process_user_time": 3.216964, "process_kernel_time": 0.48491, "process_mem_max_rss": "214204416", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m15:32:23.432595 [debug] [MainThread]: Command `dbt init` failed at 15:32:23.432426 after 730.80 seconds
[0m15:32:23.433245 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10986d250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109150790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x135e2ca90>]}
[0m15:32:23.433837 [debug] [MainThread]: Flushing usage events
[0m15:32:26.577229 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109cd9a90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109d51c90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109d52550>]}


============================== 15:32:26.579478 | 7082bed5-4fe9-44b6-8555-d809e5825ef1 ==============================
[0m15:32:26.579478 [info ] [MainThread]: Running with dbt=1.7.2
[0m15:32:26.579776 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/darshanajit/.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt init', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m15:32:35.060929 [info ] [MainThread]: A project called airflow_dbt_poc already exists here.
[0m15:32:35.066164 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 8.523191, "process_user_time": 0.796482, "process_kernel_time": 0.099412, "process_mem_max_rss": "107905024", "process_in_blocks": "0", "process_out_blocks": "0"}
[0m15:32:35.067619 [debug] [MainThread]: Command `dbt init` succeeded at 15:32:35.067300 after 8.52 seconds
[0m15:32:35.068315 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109d3c5d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109d3ddd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1050da990>]}
[0m15:32:35.069475 [debug] [MainThread]: Flushing usage events
[0m15:32:42.635942 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108bc09d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1092ddb50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1092caa50>]}


============================== 15:32:42.638101 | 140db546-78b7-459e-8e7b-89141ed961a6 ==============================
[0m15:32:42.638101 [info ] [MainThread]: Running with dbt=1.7.2
[0m15:32:42.638403 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/darshanajit/.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt init', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m15:32:53.623196 [debug] [MainThread]: Starter project path: /opt/homebrew/lib/python3.11/site-packages/dbt/include/starter_project
[0m15:32:53.637208 [info ] [MainThread]: 
Your new dbt project "airflow_dbt__poc" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m15:32:53.637869 [info ] [MainThread]: Setting up your profile.
[0m15:34:44.785209 [info ] [MainThread]: Profile airflow_dbt__poc written to /Users/darshanajit/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m15:34:44.788700 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 122.11948, "process_user_time": 2.943665, "process_kernel_time": 0.258983, "process_mem_max_rss": "214777856", "process_in_blocks": "0", "process_out_blocks": "0"}
[0m15:34:44.789784 [debug] [MainThread]: Command `dbt init` succeeded at 15:34:44.789546 after 122.12 seconds
[0m15:34:44.790727 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x14182ead0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104666810>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104666890>]}
[0m15:34:44.791485 [debug] [MainThread]: Flushing usage events
[0m15:35:02.321166 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10552ea90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1055a20d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1055a2990>]}


============================== 15:35:02.323449 | 374e5b45-1710-4f70-b8dd-aac2e1b9f472 ==============================
[0m15:35:02.323449 [info ] [MainThread]: Running with dbt=1.7.2
[0m15:35:02.323784 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': '/Users/darshanajit/.dbt', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m15:35:02.324111 [info ] [MainThread]: dbt version: 1.7.2
[0m15:35:02.324267 [info ] [MainThread]: python version: 3.11.8
[0m15:35:02.324411 [info ] [MainThread]: python path: /opt/homebrew/opt/python@3.11/bin/python3.11
[0m15:35:02.324539 [info ] [MainThread]: os info: macOS-14.4.1-arm64-arm-64bit
[0m15:35:03.241596 [info ] [MainThread]: Using profiles dir at /Users/darshanajit/.dbt
[0m15:35:03.241852 [info ] [MainThread]: Using profiles.yml file at /Users/darshanajit/.dbt/profiles.yml
[0m15:35:03.242010 [info ] [MainThread]: Using dbt_project.yml file at /Users/darshanajit/work/airflow_dbt/dbt_project.yml
[0m15:35:03.242187 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m15:35:03.242407 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 0.9552603, "process_user_time": 2.948014, "process_kernel_time": 0.230459, "process_mem_max_rss": "215171072", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m15:35:03.242594 [debug] [MainThread]: Command `dbt debug` failed at 15:35:03.242546 after 0.96 seconds
[0m15:35:03.242742 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10557ee50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1055b1fd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10091e890>]}
[0m15:35:03.242899 [debug] [MainThread]: Flushing usage events
